### PR TITLE
Adjust ChangelogOverlay appearance to match web

### LINKED ISSUE #7937: Changelog overlay colours should match osu-web
**Screenshots or videos showing encountered issue:** 
lazer:
osu!_cxjbfjrpk0

web:
chrome_ryM8rKvlQM
**osu!lazer version:** 
commit 66bd8882279679f656360efbcfbb7daf72e2260e

### PR DESCRIPTION
**Pull request by EVAST9919**
Closes https://github.com/ppy/osu/issues/7937

Current:
Screenshot_1
Screenshot_2
Screenshot_3
New:
Screenshot_4
Screenshot_5
Screenshot_6

### CODE REVIEW

File: osu.Game/Overlays/Changelog/ChangelogBuild.cs [hunk 1]
```diff
@@ -83,9 +83,9 @@ private void load(OsuColour colours)
                             {
                                 Anchor = Anchor.CentreLeft,
                                 Origin = Anchor.CentreRight,
-                                Size = new Vector2(fontSmall.Size),
+                                Size = new Vector2(10),
                                 Icon = entry.Type == ChangelogEntryType.Fix ? FontAwesome.Solid.Check : FontAwesome.Solid.Plus,
-                                Colour = entryColour,
+                                Colour = entryColour.Darken(0.7f),
```
Comment (bdach) [inline osu.Game/Overlays/Changelog/ChangelogBuild.cs]: If we're going for 100% parity, this should be
```suggestion
                                Colour = entryColour.Opacity(0.5f),
```
image

File: osu.Game/Overlays/Changelog/ChangelogBuild.cs [hunk 2]
```diff
@@ -123,9 +123,9 @@ private void load(OsuColour colours)
                         });
                     }
 
-                    title.AddText(" by ", t =>
+                    title.AddText("  by ", t =>
```
Comment (bdach) [inline osu.Game/Overlays/Changelog/ChangelogBuild.cs]: You can remove spaces altogether here and do
```csharp
t.Padding = new MarginPadding { Left = 10 };
```

File: osu.Game/Overlays/Changelog/UpdateStreamBadge.cs [hunk 3]
```diff
@@ -16,99 +14,90 @@
...
             : base(stream)
         {
-            Size = new Vector2(stream.IsFeatured ? badge_width * 2 : badge_width, badge_height);
+            this.stream = stream;
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(OverlayColourProvider colourProvider)
+        {
+            Size = new Vector2(stream.IsFeatured ? badge_width * 2 : badge_width, 60);
```
Comment (bdach) [inline osu.Game/Overlays/Changelog/UpdateStreamBadge.cs]: The height was changed from 66.5 here, and web-side it is 66.5 right now, but it was not a set pixel size as far as I can tell, it just came out like this due to sizings and `line-height: 1.25;`. We could add paddings to emulate line height but unsure if it matters that much.

File: osu.Game/Overlays/Changelog/UpdateStreamBadge.cs [hunk 4]
```diff
@@ -16,99 +14,90 @@
...
+                                    Font = OsuFont.GetFont(size: 12, weight: FontWeight.Black),
+                                },
+                                new OsuSpriteText
+                                {
+                                    Text = stream.LatestBuild.DisplayVersion,
+                                    Font = OsuFont.GetFont(size: 16, weight: FontWeight.Regular),
+                                },
+                                new OsuSpriteText
+                                {
+                                    Text = stream.LatestBuild.Users > 0 ? $"{stream.LatestBuild.Users:N0} {"user".Pluralize(stream.LatestBuild.Users == 1)} online" : null,
```
Comment (bdach) [inline osu.Game/Overlays/Changelog/UpdateStreamBadge.cs]: Can be simplified quite a bit, leveraging humanizer:
```suggestion
                                    Text = stream.LatestBuild.Users > 0 ? $"{"user".ToQuantity(stream.LatestBuild.Users, "N0")} online" : null,
```

**Review by bdach — COMMENTED** [review]
Looks mostly correct, save for some nitpicks.

**Review by bdach — APPROVED** [review]
Looks good to me. I don't think the line-height stuff is overly relevant. 👍

**Review by smoogipoo — APPROVED** [review]

**Pull request merged**